pilot/cmd/pilot-agent/main.go

```diff
@@ -196,6 +199,26 @@ var (
 
 			log.Infof("Monitored certs: %#v", certs)
 
+			if role.Type == model.Static && proxyConfig.CustomConfigFile == "" {
```

Comment:
if static and CustomConfigFile != "" shouldn't we just use CustomConfigFile here? (or is that case handled below?)

Reply:
Yes, custom config file with override static option.